pkg/scheduler/statefulset/autoscaler.go

```diff
@@ -63,7 +62,7 @@ type autoscaler struct {
 	vpodLister        scheduler.VPodLister
 	logger            *zap.SugaredLogger
 	stateAccessor     st.StateAccessor
-	trigger           chan int32
+	trigger           chan struct{}
```

Comment:
On a different PR, didnt we try, or go, with `AutoscaleTrigger`,

like:

```go
type AutoscaleTrigger struct {
	AttempScaleDown bool
	Pending         int32
}
```

now, just a struct?

Reply:
Yes, trigger is always for scale up, scale down is periodic and also pending is now performed based on the state instead of being passed around (it's easier to know it's correct)